tools/mpremote/mpremote/main.py

```diff
@@ -378,18 +410,37 @@ def main():
             do_command_expansion(args)
 
             cmds = {
```

Comment:
Probably makes sense to make this `cmds` variable global, and since it's a constant, named `_COMMANDS`.

Reply:
Nice suggestion, applied